packages/cli-main/bin/dev.js

```diff
@@ -2,7 +2,7 @@
 
 process.removeAllListeners('warning');
 
-process.env.SHOPIFY_CONFIG = "debug"
+process.env.SHOPIFY_ENV = process.env.SHOPIFY_ENV ?? "development"
```

Comment:
Changes here provide allow overriding the value when invoking the CLIs. In the case of shopifolks the override is done by `dev` by exposing the variable declared in the `dev.yml`.